src/Stripe.net/Entities/PaymentIntents/PaymentIntentNextActionWechatPayDisplayQrCode.cs

```diff
@@ -16,5 +16,17 @@ public class PaymentIntentNextActionWechatPayDisplayQrCode : StripeEntity<Paymen
         /// </summary>
         [JsonProperty("image_data_url")]
         public string ImageDataUrl { get; set; }
+
+        /// <summary>
+        /// The image_url_png string used to render QR code, can be used as &lt;img src="…" /&gt;.
```

Comment:
Are the `<`/`>` replacements easy to change? Or is this how they're supposed to be?

Reply:
Good question! C# uses XML for its comments. Unfortunately we have to encode it otherwise the documentation interpreter will think they're part of the documentation tags and interpret it as such, rendering a broken document. We can see this elsewhere in the codebase: https://github.com/stripe/stripe-dotnet/search?q=%26lt%3B